package.json

```diff
@@ -15,7 +15,8 @@
     "format-check": "prettier --list-different '**/*.{js,ts,tsx,md,less,css}'",
     "deploy-docs": "pnpm run --filter @sku-private/docs deploy",
     "release": "pnpm deploy-docs && changeset publish",
-    "version": "changeset version && pnpm install --lockfile-only"
+    "version": "changeset version && pnpm install --lockfile-only",
+    "postinstall": "node ./postinstall.js"
```

Comment:
Let's move this before `prepare` since they're both [life cycle scripts](https://docs.npmjs.com/cli/v8/using-npm/scripts?v=true#life-cycle-operation-order).